src/test/java/seedu/address/testutil/TypicalStudents.java

```diff
@@ -46,6 +46,9 @@
     public static final Student CARL = new StudentBuilder().withName("Carl Kurz").withPhone("95352563")
             .withEmail("heinz@example.com").withAddress("wall street")
             .withStudyLevel("University CS1101S").withGuardianPhone("98621032").withRelationship("Mother")
+            .addSession(
```

Comment:
One of my PR changes the method from .addSession to .addSessions instead, we might get some conflict because of that.

Reply:
ok will wait for that to merge first then